packages/editor/src/dataviews/store/private-actions.ts

```diff
@@ -139,3 +177,46 @@ export const registerPostTypeActions =
 
 		doAction( 'core.registerPostTypeActions', postType );
 	};
+
+export const registerPostTypeFields =
+	( postType: string ) =>
+	async ( { registry }: { registry: any } ) => {
+		const isReady = unlock( registry.select( editorStore ) ).isEntityReady(
+			'postType',
+			postType,
+			'fields'
+		);
+		if ( isReady ) {
+			return;
+		}
+
+		unlock( registry.dispatch( editorStore ) ).setIsReady(
+			'postType',
+			postType,
+			'fields'
```

Comment:
Should we avoid this argument and just have a global registration function for entities that register both actions and fields?

Reply:
Renamed `registerPostTypeActions` to `registerPostTypeSchema` and consolidated everything there https://github.com/WordPress/gutenberg/pull/67175/commits/3e6782ea6e469534767e8e5da641fd41698cae2e